scapy/contrib/loraphy2wan.py

```diff
@@ -43,6 +42,19 @@ def extract_padding(self, p):
         return "", p
 
 
+class FCtrl_Link(Packet):
+    name = "FCtrl_UpLink"
+    fields_desc = [BitField("ADR", 0, 1),
+                   BitField("ADRACKReq", 0, 1),
+                   BitField("ACK", 0, 1),
+                   BitField("UpClassB_DownFPending", 0, 1),
+                   BitFieldLenField("FOptsLen", 0, 4)]
+
+    # pylint: disable=R0201
```

Comment:
Please try not to include pylints commands, since it doesn't affect this repo. We would typicaly disable R0201 alltogether considering the fields format.

Reply:
Got it! It seems I don't have issues with it anymore